docs/guides/setup-lando-on-windows-with-wsl-2.md

```diff
@@ -0,0 +1,75 @@
+<YouTube url="" />
+
+Developing on Windows used to cause web developers agony and pain, but with the introduction of WSL, this is no longer quite so. WSL2 provides a near-native Linux experience for developing web applications on Windows computers.
+
+Longtime Mac or Windows Lando users will be familiar with the performance difficulties associated with file system access when using Docker in a non-linux host environment. Traditionally, running Docker on Mac or Windows requires a virtual machine environment. Sharing files across the boundary between your native OS and the Linux OS running within this virtual environment creates significant performance issues.
+
+WSL2 manages to bypass the majority of this performance penalty by never mounting your source code on a non-Linux filesystem. Here is the best (highest performance) way to setup Docker (and Lando) on a Windows 10 machine in 2021:
+
+## Prerequisites
+
+**1. Uninstall Pre-existing Docker + WSL2 Backend**
+You will want to make sure that, if you have an existing Docker or WSL setup, you are **not** using the Docker + WSL2 backend. If you have this setup, [you're going to have a bad time](https://www.youtube.com/watch?v=ynxPshq8ERo). It is very sad, very frustrating, so if you are here, do yourself a favor and uninstall the Docker Desktop WSL2 backend.
+
+**2. Make Sure Your Machine Supports Virtualization**
+You also need to ensure your machine supports virtualization (Vt-d, etc.). If you're not sure, you can [check this in Task Manager](https://www.shaileshjha.com/how-to-find-out-if-intel-vt-x-or-amd-v-virtualization-technology-is-supported-in-windows-10-windows-8-windows-vista-or-windows-7-machine). You don't need to have Windows 10 Pro like the traditional Lando on Windows setup, as WSL doesn't require it.
+
+## Get WSL2 rolling
+
+The first thing we need to do is enable WSL2. To do so...
+
+1. Open the start menu and search for "Turn Windows features on or off".
+2. In the resulting window, check 'Windows Subsystem for Linux' and 'Virtual Machine Platform' and then click 'ok'. This starts the system installed for WSL and will ask us to restart after it downloads the stuff it needs. This really only gives you the WSL1 setup. To get WSL2...
+3. Download a kernel update to get WSL2 going. see the (Microsoft docs on installing the kernel update)[https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-4---download-the-linux-kernel-update-package].
+4. After you install the kernel update, launch a powershell and run `wsl --set-default-version 2` to default to WSL 2.
+5. Head to the Microsoft store and install a Linux distro. I am going to choose [Ubuntu 18.04 LTS](https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q), since it is the most common distro for Lando users and our handy (Hyperdrive)[https://github.com/lando/hyperdrive] supports it.
```

Comment:
Is this the correct Ubuntu install on the Microsoft Store @dustinleblanc?

Reply:
I think so, but it has been a bit since I did this

Reply:
Obviously, it is also possible to install it manually: https://docs.microsoft.com/en-us/windows/wsl/install-manual
But I also did the installation via store a while ago